mobile/src/app/app.tsx

```diff
@@ -7,14 +7,18 @@ import { Provider as StoreProvider } from 'react-redux';
 
 import { globalStyles } from '~/bundles/common/styles/styles';
 import { store } from '~/framework/store/store';
-import { Root as RootNavigation } from '~/navigations/navigations';
+import {
+    Auth as AuthNavigation,
+    // Root as RootNavigation,
+} from '~/navigations/navigations';
 
 const App: FC = () => {
     return (
         <StoreProvider store={store.instance}>
             <GestureHandlerRootView style={globalStyles.flex1}>
                 <NavigationContainer>
-                    <RootNavigation />
+                    {/* <RootNavigation /> */}
+                    <AuthNavigation />
```

Comment:
All related to my comment manipulations should be in RootNavigation component, in App we just leave RootNavigation